src/plugins/saved_objects/public/save_modal/saved_object_save_modal.tsx

```diff
@@ -323,11 +329,18 @@ export class SavedObjectSaveModal extends React.Component<Props, SaveModalState>
           checked={this.state.copyOnSave}
           onChange={this.onCopyOnSaveChange}
           label={
-            <FormattedMessage
-              id="savedObjects.saveModal.saveAsNewLabel"
-              defaultMessage="Save as new {objectType}"
-              values={{ objectType: this.props.objectType }}
-            />
+            <div>
+              <FormattedMessage
+                id="savedObjects.saveModal.saveAsNewLabel"
+                defaultMessage="Save as new {objectType}"
+                values={{ objectType: this.props.objectType }}
+              />
+              {this.props.objectType === 'Visualization (VisBuilder)' ? (
```

Comment:
same as above - we should be checking the presence of an experimental flag/property being set for that object.